pkg/kubelet/dockertools/docker_manager_test.go

```diff
@@ -2062,8 +2062,8 @@ func TestCheckVersionCompatibility(t *testing.T) {
 }
 
 func TestVersion(t *testing.T) {
-	expectedVersion := "1.8.1"
-	expectedAPIVersion := "1.20"
+	expectedVersion := "1.10.3"
+	expectedAPIVersion := "1.10.4-28.gitf476348.fc23"
```

Comment:
@ingvagabund OK. I found that `TestCheckVersionCompatibility` is only checking api versions. You may want to add a new unit test to test version comparison.